Spec/Auth.swift

```diff
@@ -1748,21 +1747,18 @@ class Auth : QuickSpec {
                     }
 
                     waitUntil(timeout: testTimeout) { done in
-                        delay(tokenParams.ttl + 1.0) {
```

Comment:
To be fair once authorize is called it forces an auth every time now, so not sure this one does?  However, we still need to test initial defaults are carried forward, and any subsequent details configured are carried forward too